example/app.js

```diff
@@ -0,0 +1,90 @@
+/* global google */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ValidatedGeocomplete from '../../src/ValidatedGeocomplete';
+
+var App = React.createClass({ // eslint-disable-line
+  /**
+   * Render the example app
+   * @return {Function} React render function
+   */
+  render: function() {
+    var fixtures = [
+      {label: 'New York', location: {lat: 40.7033127, lng: -73.979681}},
+      {label: 'Rio', location: {lat: -22.066452, lng: -42.9232368}},
+      {label: 'Tokyo', location: {lat: 35.673343, lng: 139.710388}}
+    ];
+    const requiredErrorComponent = () => <div className="derp">It is required</div>,
+      notFoundErrorComponent = props => <div className="derp">Could not find {props.userInput}</div>;
+    return ( // eslint-disable-line
+      <div>
+        <ValidatedGeocomplete
+          onAfterValidate={this.onAfterValidate}
+          requiredErrorComponent={requiredErrorComponent}
+          notFoundErrorComponent={notFoundErrorComponent}
+          errorInputClassName="make-red"
+          fixtures={fixtures}
+          onFocus={this.onFocus}
+          onActivateSuggest={this.onActivateSuggest}
+          onBlur={this.onBlur}
+          onChange={this.onChange}
+          autoActivateFirstSuggest={true}
+          onSuggestSelect={this.onSuggestSelect}
+          onSuggestNoResults={this.onSuggestNoResults}
+          location={new google.maps.LatLng(53.558572, 9.9278215)}
+          radius="20" />
+      </div>
+    );
+  },
+
+  /**
+   * When the input receives focus
+   */
+  onFocus: function() {
+    console.log('onFocus'); // eslint-disable-line
+  },
+
+  onActivateSuggest: function(value) {
+    console.log('onActivateSuggest', value);
+  },
+
+  onAfterValidate: function(isValid, value) {
+    console.log('onAfterValidate isValid:', isValid);
+    console.log('onAfterValidate value:', value);
```

Comment:
Do we need `eslint-disable-line` comments on these `console.log` lines, and the one above?